rtfs_cluster/rtfs_cluster.go

```diff
@@ -13,7 +13,7 @@ import (
 // ClusterManager is a helper interface to interact with the cluster apis
 type ClusterManager struct {
 	Config *client.Config
-	Client *client.Client
+	Client client.Client
```

Comment:
Is the change to interface part of the update? This is awesome - we can use `gomock` to mock dependencies for more thorough tests at some point (been doing this for my docker client wrapper over at `ipfs-orchestrator`)

Reply:
Yes it is. Awesome I'll add that to the JIRA